genie-biogem/src/fortran/biogem_data.f90

```diff
@@ -971,54 +971,93 @@ SUBROUTINE sub_data_update_tracerrelationships()
     ! -------------------------------------------------------- !
     ! UPDATE REDFIELD RELATIONSHIPS
     ! -------------------------------------------------------- !
-    ! if NO3 is employed;
-    ! calculate alkalnity corrections associated with the formation and destruction of organic matter from NO3
-    ! otherwise, convert PO4 units to NO3 via the P:N Redfield ratio and then calculate the ALK correction from NO3
+    ! N
```

Comment:
Would say here:
Code update for PON remineralisation

Reply:
[will improve commenting in a future merge]